.gitlab/jobs/poodle.yml

```diff
@@ -33,7 +33,7 @@ gcc_10_3_1:
 intel_19_1_2_gcc_10_3_1:
   variables:
     SPEC: " ~shared +openmp +omptask +tests %intel@=19.1.2.gcc.10.3.1 ^blt@develop"
-    POODLE_JOB_ALLOC: "--time=90 --nodes=1"
+    POODLE_JOB_ALLOC: "--time=100 --nodes=1"
```

Comment:
I don’t think this will have any effect. The shared allocation defined in `.gitlab/custom-jobs-and-variables.yml` is itself limited to 40 minutes:

Reply:
Jobs shouldn’t try to override the `POODLE_JOB_ALLOC`. That’s possible on lassen, but not on machines where we share the resource. Instead, increase the shared alloc time limit.

Reply:
Unfortunately the intel job on poodle takes anywhere between 20 and 40 minutes. So when it takes longer than usual, and is not scheduled among the first jobs in the shared alloc, it fails.
I suggest `--time=60` for the shared alloc.

Reply:
My bad. I was confused since there were time limits for the other job allocations. I increased the shared allocation time.  Thank you for pointing this out.

Reply:
I was surprised myself that some jobs would override <MACHINE>_JOB_ALLOC. It turns out it’s been that way for a very long time...
The problem with that is that those jobs were not "overlapping" and therefore where not sharing resource. However, when forced to share resource, we can see that they are much slower. We might be hitting some memory limit by building in /dev/shm.
3 options:
- increase the allocated time until it fits (may not be the solution with the best throughput).
- increase the number of nodes (this is a great way to increase throughput... on ruby (thanks to the CI reservation). On poodle it may result in much longer wait for allocation).
- try without "--overlap" (which should ideally be combined with more nodes.

Reply:
The timeout problem on ruby and poodle started a couple of days ago. Before then, I don't recall seeing this. Did something change on the system?